.tp-config.json

```diff
@@ -8,5 +8,6 @@
       "description": "Sets the base URL for links and forms.",
       "required": false
     }
-  }
+  },
+  "generator": ">=0.40.1 <2.0.0"
```

Comment:
I doubt `<2.0.0`. This means that the template will work flawlessly on the next major version, despite some breaking changes that may be made to it.

Reply:
It means that this template will work with all the generator@0.40.1 and above and all the generator@1.x. There shouldn't be breaking changes affecting the template but if there are at some point we can always change this parameter. Actually, at some point after releasing v1 of Generator, we'll change this to `^1.0.0`.

Reply:
Yeah, i mean exactly this, that you claim support of `generator@1.x.y` which is not even released :smile: 
And by [semver ](https://semver.org/#summary) - `MAJOR version when you make incompatible API changes`. So, for me it's rather bold to speak that template will support next generator version...

Reply:
Yeah, I should clarify that the work we're doing right now is precisely to launch v1 in the next week. That's why I'm already putting it there.

As per [semver](https://semver.org/#spec-item-4) too: `Major version zero (0.y.z) is for initial development. Anything MAY change at any time. The public API SHOULD NOT be considered stable.` and `Version 1.0.0 defines the public API. The way in which the version number is incremented after this release is dependent on this public API and how it changes.`.

So there's no need to make a breaking change to go from major 0 to 1. You jump when you think it's stable.

Reply:
Also, the version of this package is still on 0.x. Meaning any new version might contain breaking changes.

Reply:
> launch v1 in the next week.

That's great, no concerns then

Reply:
> launch v1 in the next week.

@Tenischev I'm pretty sure @fmvilas ment `weeks` 😅

Reply:
Yes! 😄